cogs/bridge.py

```diff
@@ -43,17 +44,20 @@
 
 # Import ujson if installed for json speedup
 try:
-    import ujson as json  # pylint: disable=import-error
+    import ujson as json  # type: ignore
 except:
     pass
 
 # Import uvloop if installed for asyncio speedup
 try:
-    import uvloop  # pylint: disable=import-error
+    import uvloop  # type: ignore
 except:
     pass
 
-aiomultiprocess.set_start_method("fork")
+if platform.system() == "Windows":
+    aiomultiprocess.set_start_method("spawn")
+else:
+    aiomultiprocess.set_start_method("fork")
```

Comment:
Setting it to spawn is redundant because that's the default. Also, we could just use sys instead of importing a whole new module for it.

```suggestion
if not sys.platform == 'win32':
    aiomultiprocess.set_start_method("fork")
```

Reply:
Didn't know that's what sys was for. Ill update it.